### PR TITLE
Add a BERT model config class that will later be used to include BERT into ranking

### PR DESCRIPTION
**Pull request by thigm85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by thigm85** [comment]
@lesters Please review. I left one failing test that I want to discuss with you because the test is important and your ONNX experience might help. 

In the test, I export a sample BERT model to ONNX. I then compare the prediction made by loading the ONNX model and running it with ONNX runtime with the prediction made directly by the model that I exported. Those two predictions should be the same (up to rounding error) but they are not and I am not sure why.

**Comment by thigm85** [comment]
@oyving please review.

**Review by oyving — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def get_target_version():
     ),
     packages=setuptools.find_packages(),
     include_package_data=True,
-    install_requires=["requests", "pandas", "docker", "jinja2", "cryptography"],
+    install_requires=["requests", "pandas", "docker", "jinja2", "cryptography", "torch", "transformers", "onnxruntime"],
```
Comment (thigm85) [inline setup.py:26]: @lesters @oyving 

I feel it is overkill to add those three libraries as dependencies to pyvespa. 

* `torch` and `transformers` are required if the user wants to build applications with BERT model.
* `onnxruntime` is only used for a unit test.

Do you think that implementing something like [this](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_graph_to_onnx.py#L267-L271) is more sensible than requiring everyone to install all those libraries?

**Review by lesters — APPROVED** [review]
LGTM. With the change to using `convert_pytorch` I expect the test to work as intended.

File: setup.py (hunk 1 above)
Comment (lesters) [reply setup.py:26]: I think that is reasonable, but these are requirements for passing the suit of unit tests run in the build process?

Comment (thigm85) [reply setup.py:26]: Yes, but I can have a requirements.txt file responsible for installing the optional packages during the CI pipeline.

Comment (oyving) [reply setup.py:26]: Yeah, I think that makes sense. Make the dependencies marked as extras in the file and use requirements.txt to make sure everything is in place during testing.

You should also make sure we have tests that test the behavior if the dependencies are absent.

Comment (oyving) [reply setup.py:26]: I guess you are going to follow this guide? https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies

Comment (lesters) [reply setup.py:26]: That does sound sensible. +1

**Comment by thigm85** [comment]
@lesters @oyving Can you take a look at the last two commits. It implements the optional dependency discussed above.

**Comment by lesters** [comment]
LGTM!

**Pull request merged**